CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 ### Features
 - GAM App-Bidding: Support sub classes of GAM objects: `DFPRequest`, `GADRequest`, ...
 - MoPub App-Bidding: Support sub classes of `MPAdView` and `MPInterstitialAdController`.
+- Server Side bidding:
+  - `displayAdWithData` added to `CRBannerView`
+  - `loadAdWithDisplayData` added to `CRInterstitial`
```

Comment:
Any reason why the naming is different between banners and interstitials?

Reply:
Because the interstitial won't be displayed until `presentFromRootViewController` is called, hence the `load` method

Reply:
On second thoughts I'd rather suggest to keep the `loadAd` naming for all classes, event on banners it does not display the ad, just load its webview. It's the publisher responsibility to display the ad by making the ad view visible.

Reply:
As you wish. I will update the API on Android in this case for uniformity.

Reply:
Ok, updated to `loadAdWithDisplayData` on Android as well: https://github.com/criteo/android-publisher-sdk/pull/87/files

Reply:
Thank you, did the same [there](https://github.com/criteo/ios-publisher-sdk/compare/0d519e89680afa345ee7552a25009ab60aeb1733..e29c9c5b412acb948a10bbbe699f95b18e820ba7)